command/operator_rekey.go

```diff
@@ -58,6 +63,11 @@ Usage: vault operator rekey [options] [KEY]
   the command. If key is specified as "-", the command will read from stdin. If
   a TTY is available, the command will prompt for text.
 
+  If the flag -target=recovery is supplied, then this operation will
+  generate a new set of recovery keys rather than unseal keys. These recovery
+  keys provide an authorization mechanism for actions that require a quorum. The
+  recovery keys will not be able to unseal vault.
```

Comment:
This is actually changing in 1.13 (but only on oss), so I'd leave this out.

Reply:
Removed